frontend/src/Components/Events.js

```diff
@@ -1,19 +1,32 @@
 import React from "react";
-import { testData } from "../TestData.js";
+// import { testData } from "../TestData.js";
 import { Event } from "./Event.js";
 
 const Events = props => {
     return (
         <div class="events">
-            {testData.map(function(event) {
-                return (
-                    <Event
-                        name={event.name}
-                        description={event.description}
-                        date={event.date}
-                    />
-                );
-            })}
+            {fetch("http://localhost:3001//all-events")
```

Comment:
http://localhost:3001//all-events will not work for me, http://localhost:3001/all-events will